### ISSUE TITLE
[Adaptive Cards-FlightDetails.json]: In Scan Mode, Screen reader focus is going on some hidden element while navigating between the 'SFO' and 'JFK' on the 'FlightDetails.json' card.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9228]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0
Screen Reader: Narrator/NVDA

**Repro steps:**
1. Turn on Narrator through Ctrl + Windows + Enter key and enable scan mode.
2. Open the Adaptive Cards Visualizer application.
3. Navigate to the 'FlightDetails.json' control present on left navigation using tab key and activate it using enter key.
4. 'FlightDetails.json' adaptive card will appear on right sight of the pane.
5. Navigate in between the 'SFO' and 'JFK' using arrow key.
6. Verify that, in Scan Mode, screen reader focus is going on some hidden element while navigating between the 'SFO' and 'JFK' on the 'FlightDetails.json' card or not.

**Actual Result:**
In Scan Mode, Screen reader focus is going on some hidden element while navigating between the 'SFO' and 'JFK' on the 'FlightDetails.json' card using Up/Down arrow key. While navigating on the hidden element the screen reader announces as "Flight To" image.

**Observation:**
NVDA browse mode is not working in the application.

**Expected Result:**
In Scan Mode, Screen Reader focus should not land on the hidden place while navigating the 'FlightDetails.json' card using Up/Down arrow key. Either it should not lands on the hidden place or visually the image should be appeared.

**User Impact:**
It will impact the screen reader user as they will get confused if the focus lands on the hidden place.

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/meaningful-sequence.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/8929a4b1-d410-4dc5-a6dc-e2fe913f3bf8)

https://github.com/user-attachments/assets/ec4c58b5-32f1-4c65-bc63-9fde931521fc

### Card JSON

```JSON
N/A
```

**Comment by vagpt** [comment]
Since the product is getting deprecated, hence closing this issue.
